frontend_lib/src/tinymceAutoCompleteHelper.js

```diff
@@ -93,36 +94,35 @@ export const tinymceAutoCompleteHandleKeyDown = (event, setState, isAutoComplete
 }
 
 export const tinymceAutoCompleteHandleClickItem = (item, setState) => {
-  tinymce.activeEditor.focus()
-  const selection = tinymce.activeEditor.dom.select('span#autocomplete')[0]
-  tinymce.activeEditor.selection.select(tinymce.activeEditor.dom.select('span#autocomplete')[0])
-  tinymce.activeEditor.execCommand('mceInsertContent', false, `${item.mention}&nbsp;`)
-  tinymce.activeEditor.dom.remove(selection)
+  if (!item.mention) {
+    console.log('BUG: this member does not have a username')
+    return
+  }
 
-  setState({ isAutoCompleteActivated: false })
-}
+  const sel = tinymce.activeEditor.selection.getSel()
+  const text = sel.anchorNode.textContent
 
-export const tinymceAutoCompleteHandleSelectionChange = (selectionId, setState, isAutoCompleteActivated) => {
-  if (selectionId === 'autocomplete__searchtext' || selectionId === 'autocomplete__start') {
-    if (!isAutoCompleteActivated) setState({ isAutoCompleteActivated: true })
-    return
+  let posAt = sel.anchorOffset
+  while (posAt >= 0 && text[posAt] !== '@') {
+    posAt--
   }
```

Comment:
```suggestion
  const posAt = text.lastIndexOf('@', sel.anchorOffset)
```

Reply:
Ok, you are right. I'll adapt the code under. It would theoretically work without modification because an out of bound will give undefined and fail the check === '@', but not a fan of this.

Reply:
Err, actually no, I'd have to work on a substring of `text`. Something like:

```js
const posAt = text.substring(0, sel.anchorOffset).lastIndexOf('@')
```

Is it worth it @grignards ?

Reply:
ok, did it